docs/data-tunnel/route-types/01-tss.md

```diff
@@ -1,4 +1,4 @@
-# TSS Integration
+# TSS Route
```

Comment:
<img width="288" alt="image" src="https://github.com/user-attachments/assets/eac0df13-39dd-4e37-b5b7-951a6e278085" />

`route-types` name format is not aligned with other topics. Please align.